magicctapipe/scripts/lst1_magic/lst1_magic_event_coincidence.py

```diff
@@ -80,44 +84,89 @@
 # The final digit of timestamps
 TIME_ACCURACY = 100 * u.ns
 
-# The telescope positions used in a simulation
-TEL_POSITIONS = {
-    1: [-8.09, 77.13, 0.78] * u.m,  # LST-1
-    2: [39.3, -62.55, -0.97] * u.m,  # MAGIC-I
-    3: [-31.21, -14.57, 0.2] * u.m,  # MAGIC-II
-}
+
+def telescope_positions(config):
+    """
+    This function computes the telescope positions with respect to the array baricenter. 
+    The array can have any configuration, e.g.: M1+M2+LST1+LST2, the full MAGIC+LST array, etc.
+    
+    Parameters
+    ----------
+    config: dict
+        dictionary generated from an yaml file with information about the telescope IDs.
+        
+    Returns
+    -------
+    TEL_POSITIONS: dict
+        Dictionary with telescope positions in the baricenter reference frame of the adopted array.
+    """
+    
+    #Telescope positions in meters in a generic reference frame:
+    RELATIVE_POSITIONS = {
+    "LST-1" : [-70.930, -52.070, 43.00],
+    "LST-2" : [-35.270,  66.140, 32.00],
+    "LST-3" : [75.280 ,  50.490, 28.70],
+    "LST-4" : [30.910 , -64.540, 32.00],
+    "MAGIC-I" : [-23.540, -191.750, 41.25],
+    "MAGIC-II" : [-94.05, -143.770, 42.42]
+    }
+    
+    telescopes_in_use = {}
+    x = np.asarray([])
+    y = np.asarray([])
+    z = np.asarray([])
+    for k, v in config["mc_tel_ids"].items(): 
+        if v > 0:
+            telescopes_in_use[v] = RELATIVE_POSITIONS[k]
```

Comment:
a check if k is in RELATIVE_POSITIONS would be useful here (throwing an exception if not). this could be the case if somebody renames the telescopes in the input card or tries something non-standard like adding MSTs

Reply:
Done.